src/Number.php

```diff
@@ -1,135 +1,105 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Money;
 
+use InvalidArgumentException;
+
+use function abs;
+use function explode;
+use function is_float;
+use function is_int;
+use function ltrim;
+use function min;
+use function rtrim;
+use function sprintf;
+use function str_pad;
+use function strlen;
+use function substr;
+
 /**
  * Represents a numeric value.
  *
- * @author Frederik Bosch <f.bosch@genkgo.nl>
+ * @internal this is an internal utility of the library, and may vary at any time. It is mostly used to internally validate
```

Comment:
This class is now `@internal` and no longer part of the BC promise of this library.

That is intentional: the amount of voodoo and magic in here is NOT sustainable long-term, and we should put our efforts in trying to get rid of it in future, if possible.